website/inject.js

```diff
@@ -1,14 +1,16 @@
 const fs = require('fs')
```

Comment:
How about `const { promises: fs } = require('fs')` and then replacing `fs.promises.` with `fs.`?

Reply:
This file is also using `readFileSync` and `writeFileSync`. I suppose we could switch all of them to async version and then replace this line with `const fs = require('fs/promises')`. I'll do that in a follow up PR if that's OK.

Reply:
ah yes, sounds good!